daml/Chat/V3.daml

```diff
@@ -66,7 +65,7 @@ template UserInvitation
         do
           let contacts = M.fromList [(operator, "Operator")]
           create AddressBook with ..
-          create UserSettings with archiveMessagesAfter = 14, ..
+          create UserSettings with archiveMessagesAfter = 1209600, ..
```

Comment:
a bit odd how the units of "archiveMessagesAfter" is changing here without any other explanation.

was the field previously unused (with no units specified), therefore allowing this change to occur without other changes, despite the units of the field and the value changing?

Reply:
Yea, checking the codebase, I don't see any major references to that field. I figured change the unit to `second` can make a lot of things easier...

Reply:
ok! makes sense. it might be good to more explicitly designate the field units as seconds somehow in the code in that case.